### PR TITLE
feat(timepicker): add seconds support

### PR DESCRIPTION
**Pull request by rdinicut**

### CODE REVIEW

**Comment by Zielun** [comment]
What's the status of this feature change? Any plans to release it?

**Comment by siebeneicher** [comment]
+1

File: karma.conf.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ exclude = [
 // - Safari
 // - PhantomJS
 browsers = [
-  'Chrome'
```
Comment (wesleycho) [inline karma.conf.js]: Undo this change.

File: src/timepicker/timepicker.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ angular.module('ui.bootstrap.timepicker', [])
 .constant('timepickerConfig', {
   hourStep: 1,
   minuteStep: 1,
+  secondStep: 1,
   showMeridian: true,
+  showSeconds:true,
```
Comment (wesleycho) [inline src/timepicker/timepicker.js]: Default should be false, to agree with current behavior.

**Comment by wesleycho** [comment]
The directive should default to the current behavior, but allow for optional overriding via a config object at the directive level.

In the directive's isolate scope, we could have
<pre>
scope: {
  config: '=?'
}
</pre>
And if it is present, use it to override the default options.

**Comment by Foxandxss** [comment]
Let's continue in #4272
